starfish/types/_constants.py

```diff
@@ -40,3 +40,5 @@ class Features:
     CODE_VALUE = 'v'
     SPOT_RADIUS = 'radius'
     DISTANCE = 'distance'
+    PASSES_FILTERS = 'passes_filters'
```

Comment:
I still think this should be called PASSES_THRESHOLDS. filters get's conflated with image filtering, which I find deeply confusing.